Unable to build on Ubuntu Bionic

## Expected behavior and actual behavior.

I would like to be able to build MapServer-8.0.0-beta2 on Ubuntu Bionic. Indeed, it is required by the [repo2docker](https://repo2docker.readthedocs.io/en/latest/) tool which can be used to build this MapServer version.

I used the following configuration:

```
cmake .. \
       -DWITH_PYTHON=0 \
       -DWITH_CLIENT_WMS=1 \
       -DWITH_CLIENT_WFS=1 \
       -DCMAKE_PREFIX_PATH=/usr/ \
       -DWITH_KML=1 \
       -DWITH_OGCAPI=1 \
       -DPROTOBUF_INCLUDE_DIR=/usr/include/google//protobuf-c \
       -DPROTOBUFC_LIBRARY=/usr/lib/x86_64-linux-gnu/libprotobuf-c.so \
       -DPROTOBUF_COMPILER=/usr/bin/protoc-c \
       -DPYTHON_INCLUDE_DIRS=$(python -c "from distutils.sysconfig import get_python_inc; print(get_python_inc())")  \
       -DPYTHON_LIBRARY=$(python -c "import distutils.sysconfig as sysconfig; print(sysconfig.get_config_var('LIBDIR'))") \
       -DCMAKE_INSTALL_PREFIX=$LOCALDIR
```

Then, when trying to run ``make``, I got the following error:

```

6.415 /home/jovyan/work/workspace/mapserver-8.0.0-beta2/mapfile.c:6124:8: error: unknown type name ‘bool’
6.415  static bool msGetCWD(char* szBuffer, size_t nBufferSize, const char* pszFunctionName)
6.415         ^~~~
6.451 CMakeFiles/mapserver.dir/build.make:1845: recipe for target 'CMakeFiles/mapserver.dir/mapfile.c.o' failed

```

## Steps to reproduce the problem.

Create an account on [wholetale.org](https://wholetale.org), then use this [GitHub repository](https://github.com/GeoLabs/tb18-ms-ogc-tale) to [create a new tale](https://wholetale.readthedocs.io/en/stable/users_guide/compose.html#creating-tales-from-git-repositories). From your created tale, replace lines 13 and 14 of the [postBuild](https://github.com/GeoLabs/tb18-ms-ogc-tale/blob/main/postBuild#L13) file with the content below.

```
curl -L -o mapserver-8.0.0-beta2.tar.gz https://github.com/MapServer/MapServer/releases/download/rel-8-0-0-beta2/mapserver-8.0.0-beta2.tar.gz
tar -xvf mapserver-8.0.0-beta2.tar.gz
cd mapserver-8.0.0-beta2
```

Then, press the "Run Tale" button to get the Docker image build and the error message reported.

## Attach simple test case (drag/drop it here)

See before.

## Operating system

Ubuntu 18.04 64 bit (``FROM buildpack-deps:bionic``)

## MapServer version and installation method

MapServer-8.0.0-beta2

--- a/mapfile.c
+++ b/mapfile.c
@@ -32,6 +32,7 @@
 #endif
 
 #include <stdarg.h>
+#include <stdbool.h>
 #include <assert.h>
 #include <ctype.h>
 #include <float.h>

